main.go

```diff
@@ -167,21 +170,36 @@ func main() {
 //   3. Run data conversion
 //   4. Generate report
 func toSpanner(driver, projectID, instanceID, dbName string, ioHelper *ioStreams, outputFilePrefix string, now time.Time) error {
-	conv, err := schemaConv(driver, ioHelper)
-	if err != nil {
-		return err
-	}
-	// close the seekable file
+	conv := internal.MakeConv()
+	var err error
+	if !dataOnly {
+		conv, err = schemaConv(driver, ioHelper)
+		if err != nil {
+			return err
+		}
+		// close the seekable file
```

Comment:
Looks like a previous PR introduced a spurious new line between the comment and the code.
Suggest removing the new line.
Or we could just remove the comment -- I think it is pretty self-evident.

Reply:
Correct. Removed comment.